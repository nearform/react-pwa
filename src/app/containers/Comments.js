import { connect } from 'react-redux'

import CommentsList from '../components/CommentsList'

function mstp (state) {
  return {
    comments: state.pageLoader.data
  }
}

export default connect(mstp)(CommentsList)
