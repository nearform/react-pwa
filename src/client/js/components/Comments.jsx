import { rem } from 'csx'
import React from 'react'
import TimeAgo from 'react-timeago'
import { style } from 'typestyle'
import { debugClassName } from '../styles/common'
import { More, moreLinkClassName } from './More'

const getTitle = title => {
  return (title || '')
    .split('in "')
    .pop()
    .replace(/"/g, '')
}

const commentsListClassName = style(debugClassName('comments-list'), {
  listStyle: 'none',
  margin: 0,
  padding: '1em'
})

const commentsListItemClassName = style(debugClassName('comments-list-item'), {
  lineHeight: '10pt',
})

const commentsListTitleClassName = style(debugClassName('comments-list-title'), {
  display: 'block',
  padding: `0 0 ${rem(1)}`
})

const commentsListContentClassName = style(debugClassName('comments-list-content'), {
  color: '#000',
  display: 'block',
  padding: `0 0 ${rem(2)} ${rem(1)}`,
  $nest: {
    p: {
      margin: `${rem(0.5)} 0`
    }
  }
})

export function Comments({ data: comments, location }) {
  return (
    <React.Fragment>
      <ul className={commentsListClassName}>
        {comments && comments.map((comment, index) => (
          <li className={commentsListItemClassName} key={`${comment.isoDate}-${index}`}>
            <span className={commentsListTitleClassName}>
              {comment.creator} <TimeAgo date={comment.isoDate} />
              {' | '} on: {getTitle(comment.title)}
            </span>
            <span className={commentsListContentClassName} dangerouslySetInnerHTML={{ __html: comment.content }} />
          </li>
        ))}
      </ul>
    </React.Fragment>
  )
}
