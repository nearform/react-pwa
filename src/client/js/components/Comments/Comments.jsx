import React from 'react'
import TimeAgo from 'react-timeago'
import { classes, stylesheet } from 'typestyle'
import { loadingAnimation, placeholder } from '../../styles/common'

const getTitle = title => {
  return (title || '')
    .split('in "')
    .pop()
    .replace(/"/g, '')
}

const styles = stylesheet({
  commentsList: {
    listStyle: 'none',
    margin: 0,
    padding: '1em',
  },
  commentsListItem: {
    position: 'relative',
    overflow: 'hidden',
    lineHeight: '10pt',
  },
  commentsListTitle: {
    display: 'block',
    margin: '0 0 1em',
  },
  commentsListContent: {
    color: 'black',
    display: 'block',
    margin: '1em 0 1em 1em',
    $nest: {
      p: {
        margin: `.5em 0`,
      },
    },
  },
  noComments: {
    padding: '1em',
    textAlign: 'center',
  },
})

export function Comments({ data: comments }) {
  if (comments) {
    if (comments.length === 0) {
      return <div className={styles.noComments}>No further comments to display.</div>
    }

    return (
      <ul className={styles.commentsList}>
        {comments.map((comment, index) => (
          <li className={styles.commentsListItem} key={`${comment.isoDate}-${index}`}>
            <span className={styles.commentsListTitle}>
              <TimeAgo date={comment.isoDate} /> by {comment.creator} on {getTitle(comment.title)}
            </span>
            <span className={styles.commentsListContent} dangerouslySetInnerHTML={{ __html: comment.content }} />
          </li>
        ))}
      </ul>
    )
  }

  return (
    <ul className={styles.commentsList}>
      {Array(10)
        .fill({})
        .map((_, index) => (
          <li className={classes(styles.commentsListItem, loadingAnimation)} key={`placeholder-${index}`}>
            <span className={classes(styles.commentsListTitle, placeholder)}>time ago by placeholder</span>

            <span className={classes(styles.commentsListContent, placeholder)}>placeholder comments</span>

            <span className={classes(styles.commentsListContent, placeholder)}>placeholder comments</span>

            <span className={classes(styles.commentsListContent, placeholder)}>placeholder comments</span>
          </li>
        ))}
    </ul>
  )
}
