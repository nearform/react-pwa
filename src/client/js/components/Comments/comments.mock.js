export const commentsMock = [
  {
    content:
      'Why can’t you raise funding? If you truly have a great product with product/market fit, and established LTV greater than CAC, then you are a perfect investment for a VC who just needs to pour some gas on the fire.<p>This is exactly the kind of problem a Series A is meant to solve.</p><p>On the other hand, if the problem is your competitors are VC funded and are spending more than LTV on CAC, then (1) that sucks, but eventually they will burn out, and (2) you need to find ways to compete with more finely targeted ads that result in higher CTR to outweigh the bid differentials. Or, you need to raise the price of your product to increase LTV in line with the inflated CAC.</p>',
    creator: 'chatmasta',
    title: 'New comment by chatmasta in "America\'s Startup Scene Is Looking Anemic"',
    link: 'https://news.ycombinator.com/item?id=17256016',
    pubDate: 'Thu, 07 Jun 2018 14:29:15 +0000',
    'dc:creator': 'chatmasta',
    contentSnippet:
      'Why can’t you raise funding? If you truly have a great product with product/market fit, and established LTV greater than CAC, then you are a perfect investment for a VC who just needs to pour some gas on the fire.This is exactly the kind of problem a Series A is meant to solve.On the other hand, if the problem is your competitors are VC funded and are spending more than LTV on CAC, then (1) that sucks, but eventually they will burn out, and (2) you need to find ways to compete with more finely targeted ads that result in higher CTR to outweigh the bid differentials. Or, you need to raise the price of your product to increase LTV in line with the inflated CAC.',
    guid: 'https://news.ycombinator.com/item?id=17256016',
    isoDate: '2018-06-07T14:29:15.000Z',
  },
  {
    content:
      "&gt; This hesitation demonstrates the often tense relationship between game preservationists and private collectors. According to a screenshot uploaded by Saint, the private collector threatened to pull the entire folder of content from the directory and stop uploading games altogether if anyone leaked Labyrinthe.<p>I really can't empathize with the attitude of the private collectors.  It's extremely selfish to want to keep a work &quot;lost&quot; so that your copy is more valuable and special.  This attitude also greatly increases the chance the work will become <i>truly lost</i>, since you'll die eventually and your heirs may not realize its significance.  Why not just take credit for finding and releasing it to the world?</p>",
    creator: 'kinsomo',
    title: "New comment by kinsomo in '70 Lost Japanese Games Have Been Found in a Folder of ROMs on a Private Forum'",
    link: 'https://news.ycombinator.com/item?id=17256015',
    pubDate: 'Thu, 07 Jun 2018 14:29:02 +0000',
    'dc:creator': 'kinsomo',
    contentSnippet:
      "> This hesitation demonstrates the often tense relationship between game preservationists and private collectors. According to a screenshot uploaded by Saint, the private collector threatened to pull the entire folder of content from the directory and stop uploading games altogether if anyone leaked Labyrinthe.I really can't empathize with the attitude of the private collectors.  It's extremely selfish to want to keep a work 'lost' so that your copy is more valuable and special.  This attitude also greatly increases the chance the work will become truly lost, since you'll die eventually and your heirs may not realize its significance.  Why not just take credit for finding and releasing it to the world?",
    guid: 'https://news.ycombinator.com/item?id=17256015',
    isoDate: '2018-06-07T14:29:02.000Z',
  },
  {
    content:
      '+1, and the test seems to use php 5.5, this is definitely not OK<p>Also since we are at php, how Laravel is much faster than Symfony? and why Symfony has so many error?...</p>',
    creator: 'gregoriol',
    title: "New comment by gregoriol in 'Web Framework Benchmarks'",
    link: 'https://news.ycombinator.com/item?id=17256014',
    pubDate: 'Thu, 07 Jun 2018 14:28:59 +0000',
    'dc:creator': 'gregoriol',
    contentSnippet:
      '+1, and the test seems to use php 5.5, this is definitely not OKAlso since we are at php, how Laravel is much faster than Symfony? and why Symfony has so many error?...',
    guid: 'https://news.ycombinator.com/item?id=17256014',
    isoDate: '2018-06-07T14:28:59.000Z',
  },
  {
    content:
      'My goal has been to increase protein intake and reduce my caffeine. My body and brain works better when fueled with protein. With caffeine I have to &quot;refuel&quot; every 30 - 60 minutes and snack on stuff but with a big protein meal I find myself not thinking of food or drink for 3 hours of hard focus.',
    creator: 'ddtaylor',
    title: "New comment by ddtaylor in 'Caffeine dosing strategies to optimize alertness during sleep loss'",
    link: 'https://news.ycombinator.com/item?id=17256013',
    pubDate: 'Thu, 07 Jun 2018 14:28:50 +0000',
    'dc:creator': 'ddtaylor',
    contentSnippet:
      "My goal has been to increase protein intake and reduce my caffeine. My body and brain works better when fueled with protein. With caffeine I have to 'refuel' every 30 - 60 minutes and snack on stuff but with a big protein meal I find myself not thinking of food or drink for 3 hours of hard focus.",
    guid: 'https://news.ycombinator.com/item?id=17256013',
    isoDate: '2018-06-07T14:28:50.000Z',
  },
  {
    content:
      'To add to this, competition with the already wealthy is hard enough as-is, but when large, well-known companies have FREE offshoots in your domain, it’s almost impossible for a company to survive.<p>Can Snapchat compete with Facebook/Instagram stories? Can a startup compete with Google Analytics? Can OfferUp compete with eBay? The list goes on...</p>',
    creator: 'thosakwe',
    title: "New comment by thosakwe in 'America's Startup Scene Is Looking Anemic'",
    link: 'https://news.ycombinator.com/item?id=17256012',
    pubDate: 'Thu, 07 Jun 2018 14:28:24 +0000',
    'dc:creator': 'thosakwe',
    contentSnippet:
      'To add to this, competition with the already wealthy is hard enough as-is, but when large, well-known companies have FREE offshoots in your domain, it’s almost impossible for a company to survive.Can Snapchat compete with Facebook/Instagram stories? Can a startup compete with Google Analytics? Can OfferUp compete with eBay? The list goes on...',
    guid: 'https://news.ycombinator.com/item?id=17256012',
    isoDate: '2018-06-07T14:28:24.000Z',
  },
  {
    content:
      "&gt; Even if surveillance is already widespread, that doesn't make more surveillance a good thing.<p>I'm one of those types who uses cash, doesn't own a cell-phone, doesn't have Facebook or any of the others, opts for pat-downs at the airport as a form of protest, etc. etc.</p><p>Having security and surveillance on our ports of entry seems entirely reasonable to me.</p><p>The problem I can imagine is that we're almost guaranteed to run into No-Fly-List style mistaken-identity situations and I sincerely doubt CBP will have reasonable recourse for such mistakes.</p>",
    creator: 'mieseratte',
    title:
      "New comment by mieseratte in 'US government to use facial recognition technology at Mexico border crossing'",
    link: 'https://news.ycombinator.com/item?id=17256011',
    pubDate: 'Thu, 07 Jun 2018 14:28:23 +0000',
    'dc:creator': 'mieseratte',
    contentSnippet:
      "> Even if surveillance is already widespread, that doesn't make more surveillance a good thing.I'm one of those types who uses cash, doesn't own a cell-phone, doesn't have Facebook or any of the others, opts for pat-downs at the airport as a form of protest, etc. etc.Having security and surveillance on our ports of entry seems entirely reasonable to me.The problem I can imagine is that we're almost guaranteed to run into No-Fly-List style mistaken-identity situations and I sincerely doubt CBP will have reasonable recourse for such mistakes.",
    guid: 'https://news.ycombinator.com/item?id=17256011',
    isoDate: '2018-06-07T14:28:23.000Z',
  },
  {
    content:
      'There have been a lot of steps toward automating accounting, and most (lets say 95%) of transactions are instant and require no intervention.<p>The 5% remaining require human intervention.  In order to close the books faster you would need to either figure out a way to automate the rest (really hard) or hire a bigger finance and accounting staff who would then be idol most of the month.</p>',
    creator: 'wtvanhest',
    title: "New comment by wtvanhest in 'Short-Termism Is Harming the Economy?'",
    link: 'https://news.ycombinator.com/item?id=17256010',
    pubDate: 'Thu, 07 Jun 2018 14:28:19 +0000',
    'dc:creator': 'wtvanhest',
    contentSnippet:
      'There have been a lot of steps toward automating accounting, and most (lets say 95%) of transactions are instant and require no intervention.The 5% remaining require human intervention.  In order to close the books faster you would need to either figure out a way to automate the rest (really hard) or hire a bigger finance and accounting staff who would then be idol most of the month.',
    guid: 'https://news.ycombinator.com/item?id=17256010',
    isoDate: '2018-06-07T14:28:19.000Z',
  },
  {
    content:
      'Redistribution of wealth is bad enough to begin with, but redistributing it to pad the coffers of political patrons and harebrained unproductive finance that destroys rather than produces wealth really shows you how messed up the system can get.<p>It seems the productive end up paying immense tax and rent burdens to the unproductive tending moats and establishing political capture and networks, for the supremely undesirable service of making everything worse.</p>',
    creator: 'etherael',
    title: 'New comment by etherael in "America\'s Startup Scene Is Looking Anemic"',
    link: 'https://news.ycombinator.com/item?id=17256009',
    pubDate: 'Thu, 07 Jun 2018 14:28:06 +0000',
    'dc:creator': 'etherael',
    contentSnippet:
      'Redistribution of wealth is bad enough to begin with, but redistributing it to pad the coffers of political patrons and harebrained unproductive finance that destroys rather than produces wealth really shows you how messed up the system can get.It seems the productive end up paying immense tax and rent burdens to the unproductive tending moats and establishing political capture and networks, for the supremely undesirable service of making everything worse.',
    guid: 'https://news.ycombinator.com/item?id=17256009',
    isoDate: '2018-06-07T14:28:06.000Z',
  },
  {
    content: "I hope it's &quot;burritos&quot;. International House of Burritos? How could you possibly go wrong?",
    creator: 'mindcrime',
    title: "New comment by mindcrime in 'IHOP says it's changing name to IHOb. Huh?'",
    link: 'https://news.ycombinator.com/item?id=17256008',
    pubDate: 'Thu, 07 Jun 2018 14:28:04 +0000',
    'dc:creator': 'mindcrime',
    contentSnippet: "I hope it's 'burritos'. International House of Burritos? How could you possibly go wrong?",
    guid: 'https://news.ycombinator.com/item?id=17256008',
    isoDate: '2018-06-07T14:28:04.000Z',
  },
  {
    content: 'What does the CEO of your company make?',
    creator: 'organsnyder',
    title: "New comment by organsnyder in 'America's Startup Scene Is Looking Anemic'",
    link: 'https://news.ycombinator.com/item?id=17256007',
    pubDate: 'Thu, 07 Jun 2018 14:28:03 +0000',
    'dc:creator': 'organsnyder',
    contentSnippet: 'What does the CEO of your company make?',
    guid: 'https://news.ycombinator.com/item?id=17256007',
    isoDate: '2018-06-07T14:28:03.000Z',
  },
]
