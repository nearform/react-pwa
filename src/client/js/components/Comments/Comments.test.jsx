import React from 'react'
import { render } from 'enzyme'

import { Comments } from './Comments'

import { commentsMock } from './comments.mock'

jest.mock('react-timeago', () => () => <time>'MOCK_TIME_AGO'</time>)

it('renders correctly when no data provided', () => {
  const props = {
    data: null,
  }
  const test = render(<Comments {...props} />)
  expect(test).toMatchSnapshot()
})

it('renders correctly when empty data provided', () => {
  const props = {
    data: [],
  }
  const test = render(<Comments {...props} />)
  expect(test).toMatchSnapshot()
})

it('renders correctly when data provided', () => {
  const props = {
    data: commentsMock,
  }
  const test = render(<Comments {...props} />)
  expect(test).toMatchSnapshot()
})
