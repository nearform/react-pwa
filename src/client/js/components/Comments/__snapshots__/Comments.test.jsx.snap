// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when data provided 1`] = `
<ul
  class="commentsList_f1azx7cb"
>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by chatmasta on America's Startup Scene Is Looking Anemic
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      Why can’t you raise funding? If you truly have a great product with product/market fit, and established LTV greater than CAC, then you are a perfect investment for a VC who just needs to pour some gas on the fire.
      <p>
        This is exactly the kind of problem a Series A is meant to solve.
      </p>
      <p>
        On the other hand, if the problem is your competitors are VC funded and are spending more than LTV on CAC, then (1) that sucks, but eventually they will burn out, and (2) you need to find ways to compete with more finely targeted ads that result in higher CTR to outweigh the bid differentials. Or, you need to raise the price of your product to increase LTV in line with the inflated CAC.
      </p>
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by kinsomo on New comment by kinsomo in '70 Lost Japanese Games Have Been Found in a Folder of ROMs on a Private Forum'
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      &gt; This hesitation demonstrates the often tense relationship between game preservationists and private collectors. According to a screenshot uploaded by Saint, the private collector threatened to pull the entire folder of content from the directory and stop uploading games altogether if anyone leaked Labyrinthe.
      <p>
        I really can't empathize with the attitude of the private collectors.  It's extremely selfish to want to keep a work "lost" so that your copy is more valuable and special.  This attitude also greatly increases the chance the work will become 
        <i>
          truly lost
        </i>
        , since you'll die eventually and your heirs may not realize its significance.  Why not just take credit for finding and releasing it to the world?
      </p>
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by gregoriol on New comment by gregoriol in 'Web Framework Benchmarks'
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      +1, and the test seems to use php 5.5, this is definitely not OK
      <p>
        Also since we are at php, how Laravel is much faster than Symfony? and why Symfony has so many error?...
      </p>
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by ddtaylor on New comment by ddtaylor in 'Caffeine dosing strategies to optimize alertness during sleep loss'
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      My goal has been to increase protein intake and reduce my caffeine. My body and brain works better when fueled with protein. With caffeine I have to "refuel" every 30 - 60 minutes and snack on stuff but with a big protein meal I find myself not thinking of food or drink for 3 hours of hard focus.
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by thosakwe on New comment by thosakwe in 'America's Startup Scene Is Looking Anemic'
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      To add to this, competition with the already wealthy is hard enough as-is, but when large, well-known companies have FREE offshoots in your domain, it’s almost impossible for a company to survive.
      <p>
        Can Snapchat compete with Facebook/Instagram stories? Can a startup compete with Google Analytics? Can OfferUp compete with eBay? The list goes on...
      </p>
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by mieseratte on New comment by mieseratte in 'US government to use facial recognition technology at Mexico border crossing'
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      &gt; Even if surveillance is already widespread, that doesn't make more surveillance a good thing.
      <p>
        I'm one of those types who uses cash, doesn't own a cell-phone, doesn't have Facebook or any of the others, opts for pat-downs at the airport as a form of protest, etc. etc.
      </p>
      <p>
        Having security and surveillance on our ports of entry seems entirely reasonable to me.
      </p>
      <p>
        The problem I can imagine is that we're almost guaranteed to run into No-Fly-List style mistaken-identity situations and I sincerely doubt CBP will have reasonable recourse for such mistakes.
      </p>
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by wtvanhest on New comment by wtvanhest in 'Short-Termism Is Harming the Economy?'
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      There have been a lot of steps toward automating accounting, and most (lets say 95%) of transactions are instant and require no intervention.
      <p>
        The 5% remaining require human intervention.  In order to close the books faster you would need to either figure out a way to automate the rest (really hard) or hire a bigger finance and accounting staff who would then be idol most of the month.
      </p>
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by etherael on America's Startup Scene Is Looking Anemic
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      Redistribution of wealth is bad enough to begin with, but redistributing it to pad the coffers of political patrons and harebrained unproductive finance that destroys rather than produces wealth really shows you how messed up the system can get.
      <p>
        It seems the productive end up paying immense tax and rent burdens to the unproductive tending moats and establishing political capture and networks, for the supremely undesirable service of making everything worse.
      </p>
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by mindcrime on New comment by mindcrime in 'IHOP says it's changing name to IHOb. Huh?'
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      I hope it's "burritos". International House of Burritos? How could you possibly go wrong?
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod"
  >
    <span
      class="commentsListTitle_f1bz5gy2"
    >
      <time>
        'MOCK_TIME_AGO'
      </time>
       by organsnyder on New comment by organsnyder in 'America's Startup Scene Is Looking Anemic'
    </span>
    <span
      class="commentsListContent_ftg16dy"
    >
      What does the CEO of your company make?
    </span>
  </li>
</ul>
`;

exports[`renders correctly when empty data provided 1`] = `
<div
  class="noComments_f66kosg"
>
  No further comments to display.
</div>
`;

exports[`renders correctly when no data provided 1`] = `
<ul
  class="commentsList_f1azx7cb"
>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
  <li
    class="commentsListItem_f1y5xtod loading_f1712lg4"
  >
    <span
      class="commentsListTitle_f1bz5gy2 placeholder_fu1ixcy"
    >
      time ago by placeholder
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
    <span
      class="commentsListContent_ftg16dy placeholder_fu1ixcy"
    >
      placeholder comments
    </span>
  </li>
</ul>
`;
